### PR TITLE
Part of #17670 for file network-statistics.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation

This PR is a part of the issue #17670 Replace Typography with Text component.

In this PR I have replaced Typography with Text component in the file
**ui\components\app\edit-gas-fee-popover\network-statistics\network-statistics.js**

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, could you please update the PR description? A PR description is important because it helps other contributors to understand the purpose and context of the changes, making it easier to review and merge the PR.

**Comment by georgewrmarshall** [comment]
You also have some lint errors that need to be fixed

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall 

I am working on that.
I have finally setup ESLint (I was facing some difficulty).
Hopefully I can solve the lint issues now.

**Comment by PrgrmrHarshShukla** [comment]
I have worked on the mentioned lint issues.

@georgewrmarshall 
Kindly review it.

File: ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React, { useContext } from 'react';
 import {
   FONT_WEIGHT,
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js:3]: Hey @PrgrmrHarshShukla, we recently created enum version of the Typography helpers so you can now use that
```suggestion
  FontWeight,
```

File: ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js [hunk 2]
```diff
@@ -29,15 +29,16 @@ const NetworkStatistics = () => {
 
   return (
     <div className="network-statistics">
-      <Typography
+      <Text
         color={TextColor.textAlternative}
         fontWeight={FONT_WEIGHT.BOLD}
```
Comment (georgewrmarshall) [inline ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js:34]:
```suggestion
        fontWeight={FontWeight.Bold}
```

**Review by georgewrmarshall — COMMENTED** [review]
One suggestions based on a recent update otherwise LGTM!

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, could you please sign the [Contributor License Agreement](https://metamask.io/cla.html) prompt from GitHub?

**Review by pedronfigueiredo — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**